app/views/curation_concerns/base/_attributes.html.erb

```diff
@@ -0,0 +1,13 @@
+<table class="table table-striped <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
+  <caption class="table-heading"><h2>Work Description: <%= presenter.page_title %></h2></caption>
```

Comment:
Prepending the title with "Work Description: " in this fashion implies that the title is the work description.  The description attribute is included in the attribute_rows.